include/pnet_api.h

```diff
@@ -936,7 +937,7 @@ typedef struct pnet_im_2
  * The I&M3 data record is read-write by the controller.
  *
  * This data record is optional. If this data record is supported
- * by the application then bit 2 in the im_supported member of I&M0 shall be set.
+ * by the application then bit 3 in the im_supported member of I&M0 shall be set.
```

Comment:
Documentation for pnet_im_4_t also needs an update

Reply:
Done